pvlib/tests/iotools/test_pvgis.py

```diff
@@ -509,6 +519,12 @@ def test_get_pvgis_map_variables(pvgis_tmy_mapped_columns):
     assert all([c in pvgis_tmy_mapped_columns for c in actual.columns])
 
 
+@pytest.mark.remote_data
+def test_read_pvgis_horizon():
+    df = get_pvgis_horizon(35.171051, -106.465158)
+    elv = df.horizon_angles
+    assert all(np.isclose(elv, data_horizon_abq))
```

Comment:
Prefer to use `assert_series_equal` from our `conftest` module here (it gives better debugging info when the assertion fails).

Also should we check the other columns too?

Reply:
Changed to assert_series_equal.

We can check other columns; I just didn't want to add more big arrays to the file. I can add the azimuth column to the check as well, the other columns are useful to have when working with the horizon profile (azimuth & elevation at the solstices) but not strictly necessary.